velox/vector/fuzzer/VectorFuzzer.cpp

```diff
@@ -558,6 +558,8 @@ VectorPtr VectorFuzzer::fuzzFlat(const TypePtr& type, vector_size_t size) {
     }
 
     return fuzzRow(std::move(childrenVectors), rowType.names(), size);
+  } else if (type->isOpaque()) {
```

Comment:
Cc: @kagamiori @bikramSingh91 I remember some discussion about adding opaque support in Fuzzer; here it is :)

@kunigami could you just split the fuzzer support stuff to a different PR?

Reply:
I actually had a PR for adding it to fuzzer but I missed some codepaths that this test triggered. I'll move it to
https://github.com/facebookincubator/velox/pull/11189

Reply:
nvm, I just messed the export of the diff stack.